prestodb/hdp2.7-hive/Dockerfile

```diff
@@ -0,0 +1,93 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM centos:7
+MAINTAINER Mithun Arunan <MithunMJ@github, mithun1848@gmail.com>
+
+# Mandatory environment variables for build
+ENV HADOOP_VERSION 2.7.4
+ENV HIVE_VERSION 2.1.1
+ENV HADOOP_HOME /opt/hadoop
+ENV HIVE_HOME /opt/hive
+
+# Download hadoop
+RUN curl -s -O https://archive.apache.org/dist/hadoop/core/hadoop-$HADOOP_VERSION/hadoop-$HADOOP_VERSION.tar.gz && \
```

Comment:
Please correct if I am wrong but this image has nothing to do with HDP (Hortonworks Data Platform), it is rather apache hadoop 2.7 with hive 2.1. If so, maybe name it appropriately, like apache-2.7.

In case I am right, then what is the purpose of such image? Typically users are using Presto together with Hadoop provided by Cloudera or Hortonworks.

These images are to test Presto, are you going to use this image for that?